docroot/themes/humsci/humsci_basic/src/scss/utilities/_fonts.scss

```diff
@@ -88,13 +87,28 @@ a.hb-pill {
     }
   }
 
-  &::before {
-    border-bottom: solid hb-calculate-rems(4px);
-    @include hb-pairing-color('border-color', 'secondary');
-    content: '';
-    display: inline-block;
-    vertical-align: middle;
-    width: hb-calculate-rems(55px);
-    margin-right: hb-calculate-rems(10px);
+  @include hb-themes(('colorful', 'airy')) {
+    font-weight: hb-theme-font-weight(semibold);
+
+    &::before {
+      border-bottom: solid hb-calculate-rems(4px);
+      @include hb-pairing-color('border-color', 'secondary');
+      content: '';
+      display: inline-block;
+      vertical-align: middle;
+      width: hb-calculate-rems(55px);
+      margin-right: hb-calculate-rems(10px);
+    }
   }
+
+  @include hb-traditional {
+    font-weight: hb-theme-font-weight(regular);
+  }
+}
+
+
+// Only for use on the Traditional Theme
+.ht-serif {
```

Comment:
Even though we are only currently using a serif font for Traditional it might make sense to still use `hb-serif`. If there is a future font that has a serif option and they switch between themes then they won't have to update classes and potentially miss something. If a theme doesn't have a serif font, like Colorful, then nothing will happen when the class is applied.

Reply:
I also put the traditional mixin in there as well